### PR TITLE
Fix some more doc

### PR DESCRIPTION
**Pull request by ssnl**

### CODE REVIEW

File: torch/_tensor_docs.py [hunk 1]
```diff
@@ -1229,17 +1229,18 @@ def add_docstr_all(method, docstr):
...
-the underlying normal distribution, and not of the returned distribution:
+parameterized by the given mean :math:`\mu` and standard deviation
+:math:`\sigma`. Note that :attr:`mean` and :attr:`std` are the mean and
+standard deviation of the underlying normal distribution, and not of the
+returned distribution:
 
 .. math::
 
-    f(x) = \\dfrac{1}{x \\sigma \\sqrt{2\\pi}}\ e^{-\\dfrac{(\\ln x - \\mu)^2}{2\\sigma^2}}
+    f(x) = \dfrac{1}{x \sigma \sqrt{2\pi}}\ e^{-\frac{(\ln x - \mu)^2}{2\sigma^2}}
```
Comment (zou3519) [inline torch/_tensor_docs.py]: I don't know what \d is -- is it intentional that you removed it?

File: torch/_tensor_docs.py [hunk 2]
```diff
@@ -1746,12 +1747,14 @@ def callable(a, b) -> number
 reshape(*shape) -> Tensor
 
 Returns a tensor with the same data and number of elements as :attr:`self`,
```
Comment (zou3519) [inline torch/_tensor_docs.py]: nit: ending comma is unnecessary

File: torch/_tensor_docs.py [hunk 3]
```diff
@@ -1746,12 +1747,14 @@ def callable(a, b) -> number
...
-but with the specified shape.
+but with the specified shape. This method returns a view if :attr:`shape` is
+compatible with the current shape.
```
Comment (zou3519) [inline torch/_tensor_docs.py]: "compatible" == same shape? If so we should clarify more

**Review by zou3519 — APPROVED** [review]
Some minor comments. 

I don't understand what "compatible" means in "This method returns a view if :attr:`shape` is
compatible with the current shape.", maybe we should clarify that?

File: torch/_tensor_docs.py (hunk 1 above)
Comment (SsnL) [reply torch/_tensor_docs.py]: yep. the 2nd `d` is really ugly: https://pytorch.org/docs/master/tensors.html#torch.Tensor.log_normal_

File: torch/_tensor_docs.py (hunk 3 above)
Comment (SsnL) [reply torch/_tensor_docs.py]: I will add a pointer to the `view` doc. It is explained more over there.

File: torch/_tensor_docs.py (hunk 1 above)
Comment (zou3519) [reply torch/_tensor_docs.py]: Oh I see what it does now. Yes, remove away :)

File: torch/_tensor_docs.py (hunk 3 above)
Comment (zou3519) [reply torch/_tensor_docs.py]: (y)

**Review by soumith — APPROVED** [review]

**Pull request closed without merging**